app/src/main/java/com/xmartlabs/scasas/doapp/ui/TasksListFragment.java

```diff
@@ -132,23 +164,26 @@ private void insertTask(Task task) {
           @Override
           public void onSuccess(Task task) {
             adapter.addTask(task);
```

Comment:
add compose to check if view is still alive, and you can double check with `isViewAlive()`. Otherwise it will crash when noTaskTextView is null (view is not alive)